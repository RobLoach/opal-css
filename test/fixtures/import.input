/**
 * Example of using @import.
 */
@import "./vars.input";

body {
  background: black;
}
