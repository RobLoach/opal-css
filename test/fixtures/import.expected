.menu,.menu_link{width:200px}.menu_link{background:#056ef0}body{background:#000}
